packages/flutter/lib/src/material/chip.dart

```diff
@@ -258,7 +258,16 @@ abstract class DeletableChipAttributes {
   /// [IconThemeData.color].
   Color? get deleteIconColor;
 
+  /// Whether to use a tooltip on the delete button showing this chip's
+  /// [deleteButtonTooltipMessage].
+  ///
+  /// If null, defaults to `true`.
+  bool? get useDeleteButtonTooltip;
```

Comment:
Why is this allowed to be null?  Seems like it should just be non-nullable, and available on all the subclasses, like `autofocus` is.

Reply:
Should I add an assert in that case?

Reply:
If you remove the `?`, it will then be non-nullable. You can add an assert: it will be useful while we are still in "hybrid" mode non-nullability support, but the assert will eventually be removed, because the compiler will just prevent it from happening, and it won't be needed anymore.